src/setup/setup.h

```diff
@@ -16,7 +15,6 @@ struct UhhConfig {
 class Uhh {
     public:
         Uhh(UhhOpts& opts);
-        ~Uhh();
```

Comment:
removed because of rule of 3/5/0: https://en.cppreference.com/w/cpp/language/rule_of_three

> If a class requires a user-defined destructor, a user-defined copy constructor, or a user-defined copy assignment operator, it almost certainly requires all three. 

This class requires none.